Sources/Charcoal/Filters/Map/MapFilterViewController.swift

```diff
@@ -72,8 +92,17 @@ public class MapFilterViewController: FilterViewController {
             mapPolygonFilterViewController = nil
         }
 
+        defaultMapMode = defaultMode
+        switch defaultMapMode {
+        case .polygon:
+            selectedViewController = mapPolygonFilterViewController ?? mapRadiusFilterViewController
+        case .radius:
+            selectedViewController = mapRadiusFilterViewController
+        }
+
```

Comment:
hmm, this ended up being more complicated than it needs to be because of the computed property and needing to call the same function also later in the setup function. Right now we set the default view controller 3 times.
What if we change things a bit.
We can simply do one single thing in init: set the `selectedViewController` to whatever is `defaultMapMode`.
We can remove `defaultMapMode` variable in the class, there's no need to change it later.

Then in `setup`, we can simply change the default view controller to whatever filter is selected, but if no filter is selected, do nothing. If we do nothing there, the default view controller will still be selected. So we don't need an else there. (then we can also delete the `setMapViewToDefaultMode` function)

Does this make sense to you? 😄 
So we only need to have this logic in _one_ place where it is needed, and that is in init.

Reply:
I'll try it out! Thanks a lot, I'll close this PR and continue on the one in our main repo.